.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -56,7 +56,17 @@
            - [ ] Approve the `Approval - Partner notification` approval stage.
       1. - [ ] `Approval - Release` stage
            - [ ] **SYNC POINT**: Wait for Microsoft build release.
-           - [ ] ⚠️ 8.0: Verify the release tag in the `dnceng/dotnet-security-partners` repository matches theVMR commit to which is shown in the `Pre-Release` stage (in the `Get Associated Pipeline Run IDs` step).
+           - [ ] ⚠️ 8.0: Verify the release tag in [dotnet-security-partners](https://dev.azure.com/dotnet-security-partners/_git/dotnet) repository matches the VMR commit to which is shown in the `Pre-Release` stage (in the `Get Associated Pipeline Run IDs` step).
+           - [ ] ⚠️ If a custom tag was needed for this servicing release, mirror it to public repo.
+                - Verify that Microsoft release tag is publicly available
+                - Obtain the SHA, for which we need to create the custom tag, from `Pre-Release` stage `Get associated pipeline runs` step - follow the link to `dotnet-installer-official-ci run` to find the correct SHA.
+                - Fetch changes in internal and public `installer` repos
+                - Checkout the SHA obtained above, and create a tag with the same name as the one specified in release pipeline - double-check in `Pre-Release` stage, `Read Release info` step.
+                - Push this new tag to the public repo - this will also push all missing commit SHAs
+           - [ ] ⚠️ 6.0 / 7.0: Verify that tarball can be created without issues.
+                - This step verifies that all dependent repo SHAs have been mirrored to public repos.
+                - Checkout `dotnet/installer` at the release SHA, or custom one if used for this release.
+                - Tarball creation instructions: https://github.com/dotnet/installer#building-net-70-and-net-60
```

Comment:
Isn't this technically done by the `dotnet-installer-source-build-tarball-build` pipeline?

Reply:
All that we really need to do in this step is verify that the source tarball can be created. We don't need to actually build the source. I guess it would be nice if there was a pipeline that could be used for this step.

Reply:
This steps isn't about producing a tarball. It's about verifying that, when the release tag gets created in installer, all the associated commits for the dependent product repos (that were previously internal before the release) have been pushed to the public repos. Running the tarball creation process validates that those commits exist. If one doesn't exist, it will fail.

Reply:
> I agree that we should have a pipeline for this. I would like to see the release team run this as part of their validation. Please log an issue to track this work.

I thought there was already an existing issue for the release automation.

Reply:
> The tarball build creates the tarball via the same steps as which we would manually go through here, right?

The official installer build produces the tarball.  The tarball build just builds the tarball.

The problem here is that we don't officially release a tarball for the pre-vmr releases.  Even aside from that, anyone should be able to create a tarball from the installer release.  The release team doesn't have a process to validate the sha graph has been mirrored to public.  This is what we are trying to validate here.

Reply:
> > I agree that we should have a pipeline for this. I would like to see the release team run this as part of their validation. Please log an issue to track this work.
> 
> I thought there was already an existing issue for the release automation.

I am not finding one.